### DISCUSSION TITLE
[Feature] Handling of detected duplicate video files

### DISCUSSION THREAD

**Discussion by srosorcxisto** [immich-app/immich#10184] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Firstly, Thank you to the devs for the duplicate image feature!

Trying out the new feature, I notice that duplicate detection of video files appears to work using the generated thumbnails. The problem is that when reviewing the duplicates, the thumbnails do not provide sufficient information to confirm the duplication the same as it does for images.

For example, here are two videos with a similar thumbnails, but two different sizes. I believe one was either transcoded at some point to change the compression, or was clipped.

![Capture_blurr](https://github.com/immich-app/immich/assets/13490153/bf090e69-1de6-4735-b10b-777df45392f2)

I think this could be refined by either presenting metadata to better characterize the differences between the files such as the video length and bit-rate, or by giving the user an option to playback the file before choosing.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10173

**Reply by srosorcxisto** [reply]
> #10173

Thanks, not sure how I missed that.
